src/plugins/embeddable/public/lib/panel/panel_header/panel_actions/add_panel/open_add_panel_flyout.tsx

```diff
@@ -27,7 +27,7 @@ export async function openAddPanelFlyout(options: {
   embeddable: IContainer;
   getFactory: GetEmbeddableFactory;
   getAllFactories: GetEmbeddableFactories;
-  overlays: KibanaReactOverlays;
+  overlays: OverlayStart;
```

Comment:
with `KibanaReactOverlays` being previously the same type as  `OverlayStart` there were some typing errors in the codebase where the `KibanaReactOverlays` type was used but the actual service injected was our `Core` `OverlayStart`. Fixed them